ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -98,14 +100,22 @@ private fun Template2MainContent(state: PaywallViewState.Loaded, viewModel: Payw
 private fun IconImage(uri: Uri?) {
     uri?.let {
         Column(modifier = Modifier.widthIn(max = Template2UIConstants.maxIconWidth)) {
-            RemoteImage(
-                urlString = uri.toString(),
-                modifier = Modifier
-                    .aspectRatio(ratio = 1f)
-                    .widthIn(max = Template2UIConstants.maxIconWidth)
-                    .clip(RoundedCornerShape(Template2UIConstants.iconCornerRadius)),
-                contentScale = ContentScale.Crop,
-            )
+            // TODO-PAYWALLS: test this
+            if (uri.toString().startsWith("android.resource://")) {
+                Image(
+                    painter = painterResource(id = uri.toString().substringAfterLast("/").toInt()),
+                    contentDescription = null,
+                )
+            } else {
+                RemoteImage(
+                    urlString = uri.toString(),
+                    modifier = Modifier
+                        .aspectRatio(ratio = 1f)
+                        .widthIn(max = Template2UIConstants.maxIconWidth)
+                        .clip(RoundedCornerShape(Template2UIConstants.iconCornerRadius)),
```

Comment:
Sorry I think I commented on the commit instead of the PR.
We need to apply these to the `Image` above too.

Reply:
Oh right, good catch